src/TodoListItem.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 
-function TodoListItem( {item} ){
+function TodoListItem( {item, onRemoveTodo} ){
 
+	//Add a button element, type "button", inside the list item with text "Remove"
 	return (
 		<div>
-			<li>{item.title}</li>
+			{/* Add an onClick prop to the button element and pass a function that calls onRemoveTodo from props with the current item id as an argument */}
+			{/* Note: the onRemoveTodo needs to be wrapped in an anonymous function b/c onClick only accepts handlers */}
+			<li>
+				{item.title}<button type="submit" onClick={() => onRemoveTodo(item.id)} > Remove </button>
+			</li>
 		</div>
 	)
 
```

Comment:
JSX experessions are limited to a single expression.  Semi-colons are used to seperate expressions, so they aren't allowed.  Think about there being an implied `return` and the very beginning.  So you can't have { return 1 + 1; 2 + 2}